workshops.md

```diff
@@ -3,42 +3,56 @@ layout: page
 permalink: /workshops/
 ---
-## Workshop #1 - SF Edition
-Tuesday, July 17
-<br>Square - San Francisco, CA
-<br>1-2 engineers from SF Bay Area Bitcoin companies
-
-# Agenda:
-12:30-1:00 Introductions
-<br>1:00-2:00 Topic #1: Coin selection
-<br>2:00-2:15 Break
-<br>2:15-3:15 Topic #2: Fee estimation, RBF, CPFP
-<br>3:15-3:30 Break
-<br>3:45-4:45 Topic #3: Optech community and communication
-<br>4:45-5:00 Wrap up
-<br><br>5:30-7:30 Dinner
-
-# Format:
-- Each topic will be a roundtable discussion in which every participant has an equal opportunity to engage.
-- There will be a moderator and notetaker. The moderator will be responsible for a brief introduction of a topic and keeping discussion on track and on time.
-- We want participants to be comfortable to speak freely, so it is intended that notes and action items taken will be distributed to participants but not beyond. Participants are free to share discussion details internally at their companies and publicly, but should refrain from attributing any particular statement to a given individual (Chatham House Rules).
-
-# Possible Topic Discussion Points
-
-Topic #1: Coin selection
-- What do you do today?
-- What constraints do you face?
-- How do you prioritize potentially competing factors: blockchain efficiency, transaction throughput, privacy, minimizing fees?
-- What are best practices?
-
-Topic #2: Fee estimation, RBF, CPFP best practices
-- What do you do today?
-- What constraints do you face?
-- What are the UX implications for your customers?
-- What are best practices?
-
-Topic #3: Optech community and communication
-- What level of involvement would you like to have creating best practices open source cookbooks?
-- What do you want to see in an Optech community newsletter?
-- How do you want to engage with each other moving forward? Online forum? Slack? Dinners?
 
+Bitcoin Optech will run a series of workshops to bring Bitcoin engineers
+together to discuss approaches and challenges in implementing scaling
+technologies. Each workshop will be tailored to the member companies attending
+and the specific scaling challenges that they are facing.
+
+- Topics are discussed in a roundtable format in which every participant has an
+  equal opportunity to engage.
+
+- Each topic has a moderator and notetaker. The moderator is responsible
+  for a brief introduction of a topic and keeping discussion on track and on
+  time.
+
+- We want participants to be comfortable to speak freely. Notes and action
+  items are distributed to participants but not beyond. Participants are
+  free to share discussion details internally at their companies and publicly,
+  but should refrain from attributing any particular statement to a given
+  individual (Chatham House Rules).
+
+## Workshop #2 - Paris, Fall 2018
+
+We are in the early stages of planning the second Bitcoin Optech workshop in
+Paris for Fall 2018. Member companies who would like to attend should
+[contact us][optech email]!
```

Comment:
Looks like this isn't rendering properly; may need to add the `optech email` footnote def?

Reply:
Nevermind! Should've tACK'd.

Reply:
Nah, this is fine (tested preview); that link is defined in references.md.  The problem was the image added in last newsletter doesn't have an alt tag defined, so all the builds based on master are failing.  I'll PR a fix for that.